src/back/space/db/migrations/V0001__Init.sql

```diff
@@ -0,0 +1,39 @@
+CREATE SCHEMA IF NOT EXISTS space;
+
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE space.space (
+	id UUID PRIMARY KEY DEFAULT UUID_GENERATE_V4(),
+	name TEXT NOT NULL,
+	key TEXT NOT NULL,
```

Comment:
Давай сделаем фиксированной длинной 
```suggestion
	key VARCHAR(32) NOT NULL,
```

Reply:
Готово.